tidy3d/web/webapi.py

```diff
@@ -192,7 +192,8 @@ def monitor(task_id: TaskId) -> None:
 
     console = Console()
 
-    with console.status(f"[bold green]Working on '{task_name}'...", spinner="runner") as status:
+    with console.status(f"[bold green]Working on '{task_name}'...", spinner="runner"):
+        console.log(f"status = {status}")
```

Comment:
does this fix the "queued not showing up" issue?

Reply:
No...